server/plugin.go

```diff
@@ -122,11 +122,11 @@ func (p *Plugin) GetSyncNotifications() bool {
 }
 
 func (p *Plugin) GetSyncDirectMessages() bool {
```

Comment:
I know it's not part of this PR pre se but I'd like to have those renamed to something more accurate.